src/engraving/libmscore/beam.h

```diff
@@ -48,28 +50,27 @@ class Beam final : public Element
     Q_GADGET
     QVector<ChordRest*> _elements;          // must be sorted by tick
     QVector<mu::LineF*> beamSegments;
-    Direction _direction;
+    Direction _direction    { Direction::AUTO};
 
-    bool _up;
-    bool _distribute;                     // equal spacing of elements
-    bool _noSlope;
+    bool _up                { true };
+    bool _distribute        { false };                    // equal spacing of elements
 
-    bool _userModified[2];                // 0: auto/down  1: up
-    bool _isGrace;
-    bool _cross;
+    bool _userModified[2]   { { false } };                // 0: auto/down  1: up
```

Comment:
dropping the extra pair of braces should fix the build issue on Linux.
It only sets the 1st element to `false`, but then implicitly initializes the 2nd to `0`, which incidentially is `false`;-)